op-node/rollup/derive/calldata_source.go

```diff
@@ -143,14 +143,15 @@ func DataFromEVMTransactions(config *rollup.Config, daCfg *rollup.DAConfig, batc
 // decodeETHData will decode the data retrieved from the EVM, this data
 // was previously posted from op-batcher and contains the block height
 // with transaction index of the SubmitPFD transaction to the DA.
-func decodeETHData(celestiaData []byte) (int64, int64, error) {
+func decodeETHData(celestiaData []byte) (int64, int32, error) {
```

Comment:
This should match batcher, which is serializing as `int64, uint32`.

https://github.com/celestiaorg/optimism/pull/3/files#diff-1fc10714a3fe0349447d62b700894474ab8786e6d28bbeea56d017bfd137b676R171